### ISSUE TITLE
Another intermittent error using opencloud on Rackspace servers

### ISSUE THREAD

**Issue by mrpdean** [rackspace/php-opencloud#604]
@ycombinator   Hi ycombinator, a while ago you helped me with this issue which turns out to be a problem on the particular flavour of Rackspace server I was using:  https://github.com/rackspace/php-opencloud/issues/501#issuecomment-69525380

Everything was running smoothly for a while until recently when I started getting a slightly different error each night in my script which uploads/deleted images in CloudFiles.

Like before, the issue only occurs intermittently, and like before I've only seen the error on our Rackspace server.  I've yet to see it occur on my local VirtualBox.

The error I'm now seeing looks like this (where the URL for the image changes depending on when it bombs out):

[Guzzle\Http\Exception\ClientErrorResponseException]
  Client error response
  [status code] 404
  [reason phrase] Not Found
  [url] https://snet-storage101.syd2.clouddrive.com/v1/MossoCloudFS_8c3cef2b-4915-4d8e-8c39-4633425e3bee/vc-web/bookeasy/0001/01/thumb_567_boo
  keasy_small.jpeg

Would greatly appreciate your assistance again.

**Comment by jamiehannaford** [comment]
@mrpdean Could you send the following to sdk-support@rackspace.com, please?

1. a list of affected files, including their container names
2. the code you're using (so I can eliminate basic problems like race conditions)
3. your tenant or customer ID

Once I have these, I'll begin to investigate and pass on to our Cloud Files team who can check the logs.

**Comment by mrpdean** [comment]
Thanks @jamiehannaford 

I'm just running a few more tests on a new RS server instance to try to narrow down the cause a bit more.  There are a lot of parts at play e.g. Symfony, Sonata Media Bundle, our own Rackspace Filesystem Adapter for the Sonata Media Bundle, Rackspace PHP OpenCloud, Guzzle etc

I'll let you know how the tests go.

Thanks again

**Comment by mrpdean** [comment]
Hi @jamiehannaford 

I wasn't able to find the cause of the issue. It was occurring on a new server instance however it might not have anything to do with the server though.  I tried rolling back several bundles to previous, stable versions but the problem still persisted.

However, I have since discovered Rackspaces new CDN feature and have swapped over to that and so far I'm loving it. Not having to push thousands of images up to CloudFiles make the whole process much simpler.

This can be closed.

Thanks for your assistance.
